week06_policy_based/a2c-optional.ipynb

```diff
@@ -25,7 +27,7 @@
      "    os.system('apt-get install -y python-opengl ffmpeg')\n",
      "    os.system('pip install pyglet==1.2.4')\n",
      "    # Update the gym environment to be compatible with the atari environment.\n",
-     "    os.system('pip install gym[atari,accept-rom-license]==0.21.0')\n",
+     "    os.system('pip install gym[atari,accept-rom-license]==0.18.3')\n",
```

Comment:
Downgrading the gym allows us to use `gym.monitoring` and produce videos, unlike the version that I inserted before, which has the module but has an issue making the videos. It was creating the JSON files only. 

I had this issue in [week06/a2c-optional.ipynb](https://github.com/yandexdataschool/Practical_RL/blob/master/week06_policy_based/a2c-optional.ipynb), but I didn't know the reason.

Reply:
Strange — `gym.wrappers.monitoring.video_recorder` is in the latest `master` in the gym repo: https://github.com/openai/gym/blob/6a04d49722724677610e36c1f92908e72f51da0c/gym/wrappers/monitoring/video_recorder.py

Additionally, the following code runs fine in Colab (and prints `0.25.2`):

```python
import gym
print(gym.__version__)

from gym.wrappers.monitoring.video_recorder import VideoRecorder
```

Perhaps we could avoid downgrading `gym`?

Reply:
I will investigate that, and I will let you know once I test it out. My Biggest concern right now is the problem with the rewards in both Week06 and Week08 as I mentioned in one of the opened issues by you!

Reply:
Hello @dniku ,
    I hope you are doing well. First of all, thank you for expanding my horizon and finding a new way of recording than the one I learned with Yandex. Since the latest version of `gym` doesn't support using `Monitor`, I had to change a little bit in the workflow of the assignment to be compatible with the `gym`'s current version. I will also have to update other things with the workflow, such as the `reward_scaling` and so on. I managed to bring a score of **14.7k** with the required model's architecture by adding `BatchNorm`.
    
Thank you again! I will update the notebook, now!